### ISSUE TITLE
Remove 1.8 class files warning

### ISSUE THREAD

**Issue by lucaswerkmeister** [ceylon/ceylon-compiler#2163]
Java 7 has reached end of life, and there aren’t any known Java 8-specific issues, so I suggest we remove the following warning:

```
warning: It looks like you are using class files from a Java newer than 1.7.
  Everything should work well, but if not, let us know at https://github.com/ceylon/ceylon-compiler/issues.
  In the near future, the Ceylon compiler will be upgraded to handle Java 1.8.
```

because it’s now obsolete.

**Comment by gavinking** [comment]
+1

**Comment by luolong** [comment]
+1

**Comment by FroMage** [comment]
Well, I could remove it, but I'd have to replace it with "actually you can't compile to Java 8 bytecode so you can't use default methods"

**Comment by FroMage** [comment]
Moving to 1.3.

**Comment by gavinking** [comment]
Huh?!

I think we should remove this warning. It's very annoying and useless.

**Comment by lucaswerkmeister** [comment]
I agree. Being unable to use default methods is just another interop problem, there’s no need to warn people about it every time they compile anything at all (*even if it’s pure Ceylon code*).

**Comment by gavinking** [comment]
Or, y'know, give them an error _if and when they try to call a `default` method_.

**Comment by gavinking** [comment]
I mean, a problem with the current warning is after seeing only about `1k` times, I still didn't know I couldn't call `default` methods in Ceylon.

**Comment by FroMage** [comment]
Well, I did suggest to change the error message, but nobody answered.

**Comment by jvasileff** [comment]
+1 for eliminating the message.

The impression I got from this warning when I first saw it was 180º off - that Ceylon might be stale and lacking momentum.

I think it's fine letting folks discover that `ceylon compile` can't compile newer *Java* sources on their own, without the bad marketing and compromised UI.

**Comment by lucaswerkmeister** [comment]
Can we please remove this warning? “Everything should work well, but if not, let us know” sounds like a cover-our-asses statement because we didn’t test whether we work on 8, which isn’t true at all – by now, we’re very sure that everything we support also works on Java 8. A warning that we don’t support lambdas would be more fitting on an `import java.base "8"` statement in my opinion. (Though then it would suck that you can’t import `java.base` without warnings.)

**Comment by FroMage** [comment]
OK fine.

**Comment by FroMage** [comment]
Done, halleluia

**Comment by lucaswerkmeister** [comment]
:pray:
